augur/curate/format_dates.py

```diff
@@ -36,11 +36,13 @@ def register_parser(parent_subparsers):
 
     optional = parser.add_argument_group(title="OPTIONAL")
     optional.add_argument("--expected-date-formats", nargs="+", action="extend",
-        default=DEFAULT_EXPECTED_DATE_FORMATS,
-        help="Expected date formats that are currently in the provided date fields, " +
-             "defined by standard format codes as listed at " +
-             "https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes. " +
-             "If a date string matches multiple formats, it will be parsed as the first matched format in the provided order.")
+        help=f"""Custom date formats for values in the provided date fields, defined by standard
+                 format codes available at
+                 <https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes>.
+                 If a value matches multiple formats, it will be parsed using the first match. The
+                 following formats are automatically used: {", ".join(repr(x).replace("%", "%%") for
+                 x in BUILTIN_DATE_FORMATS)}. User-provided values will be considered
+                 before the predefined formats.""")
```

Comment:
```suggestion
                 following formats are builtin and automatically used: {", ".join(repr(x).replace("%", "%%") for
                 x in BUILTIN_DATE_FORMATS)}. User-provided values are considered
                 before the builtin formats.""")
```

Use "builtin" to lay the groundwork for the connection to `--no-builtin-date-formats` later (#1707).

Reply:
Updated:

https://github.com/nextstrain/augur/blob/89950c8881a76bde6565d69ffb807816dce3b4ab/augur/curate/format_dates.py#L43-L45